generators/client/index.js

```diff
@@ -76,6 +76,35 @@ module.exports = class extends BaseGenerator {
         }
     }
 
+    getHipster(baseName = this.baseName) {
```

Comment:
can we move this method to generator-base

Reply:
Sure. Had that first but as it is only relevant for the client generator I moved to to the client generator.

Reply:
problem is yeoman will try to run it as a phase so base is better

Reply:
:heavy_check_mark: Change it locally already. Going to use svgs too.